### PR TITLE
Add Worldofplayers.de

### LINKED ISSUE #13497: Ruleset for worldofplayer.de
Type: new ruleset
Domain: www.forum.worldofplayers.de

### PR DESCRIPTION
**Pull request by DavidLiedke**
fix #13497

### CODE REVIEW

File: src/chrome/content/rules/worldofplayers.de.xml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+	dlord.worldofplayers.de
+	dragonage.worldofplayers.de
+	farcry.worldofplayers.de
+	forenarchiv.worldofplayers.de
+	lego.worldofplayers.de
+	messiah.worldofplayers.de
+	rpb.worldofplayers.de
+	witcher.worldofplayers.de
+	wogen.worldofplayers.de
+	wotor.worldofplayers.de
```
Comment (J0WI) [inline src/chrome/content/rules/worldofplayers.de.xml]: Wildcard DNS record.

Comment (DavidLiedke) [reply src/chrome/content/rules/worldofplayers.de.xml]: They have no wildcard certificate. 

Or do you want a info like this: "Invalid Certificate - All other targets" ?

Comment (J0WI) [reply src/chrome/content/rules/worldofplayers.de.xml]: You can just add a comment that they use a wildcard DNS record. That's just a hint that nobody starts to add random hosts to the list.

Comment (DavidLiedke) [reply src/chrome/content/rules/worldofplayers.de.xml]: But i can´t find there a wildcard dns record. All targets have their own certificates.

Comment (J0WI) [reply src/chrome/content/rules/worldofplayers.de.xml]: See e.g. `foobar.worldofplayers.de`

**Comment by DavidLiedke** [comment]
Done

**Comment by J0WI** [comment]
Thanks!

**Pull request merged**